docbook-xml.yaml

```diff
@@ -0,0 +1,316 @@
+package:
+  name: docbook-xml
+  version: "4.5"
+  epoch: 0
+  description: A widely used XML scheme for writing documentation and help
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: MIT
+  dependencies:
+    runtime:
+      - libxml2-utils
+      - busybox
+  scriptlets:
+    post-install: |
+      #!/bin/busybox sh
```

Comment:
Instead of having a scriptlet here, lets merge docbook-xml and docbook-xsl into a single package and then build /etc/xml/catalog at build time.